src/platform/site-wide/sass/consolidated-patches.scss

```diff
@@ -24,6 +36,18 @@ header.merger {
       background-image: none;
     }
   }
+  .va-notice--banner-inner {
+    width: $teamsite-width;
+  }
+  #mega-menu .vetnav-panel {
```

Comment:
These menu widths might break with the updates I have in a [open PR](https://github.com/department-of-veterans-affairs/vets-website/pull/8754) now. There will be two different widths for the menu, depending on the current screen size.

Reply:
Thanks @jbalboni, tablet styles have been added to hide the third megamenu panel, and the larger screen styles have been wrapped in a large screen media query. Thanks for any additional insights on how to present these.